air/src/execution_step/instructions/call/prev_result_handler.rs

```diff
@@ -151,11 +151,11 @@ fn update_state_with_service_result<'i>(
     Ok(())
 }
 
-fn handle_service_error<'i>(
```

Comment:
Was it made by accident? Anyway, I suggest making a separate PR for such changes to keep updates consistent.

Reply:
If it was caused by updated clippy warnings, it would break master if not updated.